### PR TITLE
Updated application icons and project banner.

### PR DESCRIPTION
**Pull request by KernelGhost**
This pull request introduces a 'WinApps' project banner to the README and updates the icons for officially supported applications. While many of these icons are in the Public Domain, some are protected trademarks in certain jurisdictions. However, this situation is similar to what we currently face with the existing set of application icons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-# WinApps
-*The WinApps project, forked from Fmstrat's [original repository](https://github.com/Fmstrat/winapps).*
+<p align="center"><img align="center" width="700" src="./icons/banner_dark.svg#gh-dark-mode-only"/></p>
+<p align="center"><img align="center" width="700" src="./icons/banner_dark.svg#gh-light-mode-only"/></p>
+<hr>
 
-Run Windows applications (including [Microsoft 365](https://www.microsoft365.com/) and [Adobe Creative Cloud](https://www.adobe.com/creativecloud.html)) on GNU+Linux with `KDE` or `GNOME`, integrated seamlessly as if they were native to the OS.
+Run Windows applications (including [Microsoft 365](https://www.microsoft365.com/) and [Adobe Creative Cloud](https://www.adobe.com/creativecloud.html)) on GNU/Linux with `KDE` or `GNOME`, integrated seamlessly as if they were native to the OS.
```
Comment (oskardotglobal) [inline README.md:5]: Stallman would like a word with you.

**Review by oskardotglobal — APPROVED** [review]
I like it.

**Comment by oskardotglobal** [comment]
Please resolve the merge conflicts in order for this to be merged.
The use of the icons should be fine under Fair use in most jurisdictions.

**Comment by KernelGhost** [comment]
All merge conflicts have been resolved. Please review the changes and let me know if any further adjustments are needed. Thanks!

**Review by oskardotglobal — APPROVED** [review]
Alright, merging.

**Pull request merged**